.eslintrc.js

```diff
@@ -1,10 +1,27 @@
+const requiredSVGAttrs = Object.entries({
+  xmlns: 'http://www.w3.org/2000/svg',
+  width: 24,
+  height: 24,
+  viewBox: '0 0 24 24',
+  fill: 'none',
+  stroke: 'currentColor',
+  'stroke-width': 2,
+  'stroke-linecap': 'round',
+  'stroke-linejoin': 'round',
+}).map(([attr, value]) => ({
+  tag: 'svg',
+  attr,
+  value: String(value),
+}));
```

Comment:
importing from https://github.com/lucide-icons/lucide/blob/main/scripts/render/default-attrs.json would be better.
```suggestion
import DEFAULT_ATTRS from './scripts/render/default-attrs.json' assert { type: 'json' };
const requiredSVGAttrs = Object.entries(DEFAULT_ATTRS)
  .map(([attr, value]) => ({ tag: 'svg', attr, value: String(value) }));
```